### PR TITLE
[EDA-690] Manage history for a taxpayer

### PR DESCRIPTION
**Pull request by NahuelValencia-eb**
When a user modifies the taxpayer, these changes are recorded in history tables.

### CODE REVIEW

File: supplier_app/tests/test_view.py [hunk 1]
```diff
@@ -1601,3 +1603,173 @@ def test_taxpayer_creation_error_should_display_error_message(self):
...
+            'address_id': address.id,
+        }
+        
+        ADDRESS_POST = {
+            'address_form-street': 'San Martin',
+            'address_form-number': '21312',
+            'address_form-zip_code': '123',
+            'address_form-city': 'Mendoza',
+            'address_form-state': 'Mendoza',
+            'address_form-country': 'Argentina',
```
Comment (leandro-eb) [inline supplier_app/tests/test_view.py]: Could we use address factory to populate this?

File: supplier_app/tests/test_view.py [hunk 2]
```diff
@@ -1601,3 +1603,173 @@ def test_taxpayer_creation_error_should_display_error_message(self):
...
+                'taxpayer-history',
+                kwargs={'pk': self.taxpayer.id}
+                ),
+        )
+
+        self.assertContains(response, old_name)
+        self.assertContains(response, new_name)
+        self.assertEqual(response.template_name[0], 'AP_app/taxpayer-history-list.html')
+
+    def test_history_address_should_contain_old_and_new_values(self):
```
Comment (leandro-eb) [inline supplier_app/tests/test_view.py]: Seems like we are testing storing all results, but without a way to identify what's new and what's old. Could we check this?

File: supplier_app/tests/test_view.py [hunk 3]
```diff
@@ -1601,3 +1603,173 @@ def test_taxpayer_creation_error_should_display_error_message(self):
...
+
+        self.assertContains(response, old_street)
+        self.assertContains(response, new_street)
+        self.assertEqual(response.template_name[0], 'AP_app/taxpayer-history-list.html')
+
+    def test_history_bank_account_should_contain_old_and_new_values(self):
+        bank_account = BankAccountFactory(
+            taxpayer=self.taxpayer,
+            bank_account_number="132456789",
+            )
```
Comment (leandro-eb) [inline supplier_app/tests/test_view.py]: nit, check indentation

File: supplier_app/urls.py [hunk 4]
```diff
@@ -2,17 +2,18 @@
 
 from supplier_app.views import (
     ApTaxpayers,
+    change_taxpayer_status,
     CompanyCreatorView,
     CompanyListView,
+    company_invite,
+    company_join,
```
Comment (leandro-eb) [inline supplier_app/urls.py]: Are we using this?

File: templates/AP_app/taxpayer-history-list.html [hunk 5]
```diff
@@ -0,0 +1,102 @@
+{% extends 'base.html' %}
+{% load i18n %}
+
+
+{% block content %}
+{% if is_AP %}
+{% include 'AP_app/navbar.html' %}
+{% else %}
+{% include 'AP_app/navbar.html' %}
+{% endif %}
```
Comment (leandro-eb) [inline templates/AP_app/taxpayer-history-list.html]: This is weird, what are we checking here?

File: templates/invoices_app/history-list.html [hunk 6]
```diff
@@ -6,7 +6,7 @@
 {% if is_AP %}
 {% include 'AP_app/navbar.html' %}
 {% else %}
-{% include 'supplier_app/navbar.html' %}
+{% include 'AP_app/navbar.html' %}
 {% endif %}
```
Comment (leandro-eb) [inline templates/invoices_app/history-list.html]: Same comment as before

File: supplier_app/urls.py (hunk 4 above)
Comment (NahuelValencia-eb) [reply supplier_app/urls.py]: Yes, in some urls

**Pull request merged**